SECURITY.md

```diff
@@ -0,0 +1,16 @@
+# Security Policy
+
+We're big believers in protecting your privacy and security. As a company, we not only have a vested interest, but also a deep desire to see the Internet remain as safe as possible for us all.
+
+So, needless to say, we take security issues very seriously.
+
+In our opinion, the practice of 'responsible disclosure' is the best way to safeguard the Internet. It allows individuals to notify companies like Librosa of any security threats before going public with the information. This gives us a fighting chance to resolve the problem before the criminally-minded become aware of it.
+
+Responsible disclosure is the industry best practice, and we recommend it as a procedure to anyone researching security vulnerabilities.
+
+## Reporting a Vulnerability
+
+If you have discovered a vulnerability in this open source project or another serious security issue, do not submit it under a public issue,
+please submit it to the following.
```

Comment:
I'd prefer to set up private vulnerability reporting on github for this, rather than route things to an email address.

Reply:
I've enabled this feature now - we can link to https://github.com/librosa/librosa/security/advisories/new